testsuite/forge-cli/src/main.rs

```diff
@@ -480,16 +480,19 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
         // maximizing number of rounds and epochs within a given time, to stress test consensus
         // so using small constant traffic, small blocks and fast rounds, and short epochs.
         "consensus_stress_test" => config
-            .with_network_tests(&[&ContinuousProgressTest { target_tps: 100 }])
+            .with_network_tests(vec![&ContinuousProgressTest { target_tps: 100 }])
             .with_initial_validator_count(NonZeroUsize::new(10).unwrap())
             .with_genesis_helm_config_fn(Arc::new(|helm_values| {
                 helm_values["chain"]["epoch_duration_secs"] = 60.into();
             }))
             .with_node_helm_config_fn(Arc::new(|helm_values| {
-                helm_values["consensus"]["max_block_txns"] = 50.into();
-                helm_values["consensus"]["round_initial_timeout_ms"] = 500.into();
-                helm_values["consensus"]["round_timeout_backoff_exponent_base"] = 1.0.into();
-                helm_values["consensus"]["quorum_store_poll_count"] = 1.into();
+                helm_values["validator"]["config"]["consensus"]["max_block_txns"] = 50.into();
```

Comment:
so it doesn't work before?

Reply:
It wasn't applying these (i.e. running round fast). But was still passing as a test